packages/resource-deployment/scripts/upload-files.sh

```diff
@@ -24,6 +30,15 @@ uploadFileBatch() {
     az storage blob upload-batch --account-name "$storageAccountName" --destination "$destinationContainer" --source "$pathToSource" --pattern "$includePattern" 1>/dev/null
 }
 
+uploadFile() {
```

Comment:
call this function uploadFile & rename uploadFileBatch to uploadFiles

Reply:
updated.